docs-2.0/nebula-dashboard/2.deploy-dashboard.md

```diff
@@ -1,6 +1,6 @@
 # Deploy Dashboard
 
-The deployment of Dashboard involves five services. This topic will describe how to deploy Dashboard in detail.
+The deployment of Dashboard involve five services. This topic will describe how to deploy Dashboard in detail.To download and compile the latest source code of Nebula Dashboard, follow the instructions on the [nebula dashboard GitHub page](https://github.com/vesoft-inc/nebula-dashboard).
```

Comment:
It should be "involves". A blank space is missing between the 2nd and the 3rd sentences.